frontend/src/pages/AuthPage/index.jsx

```diff
@@ -31,14 +31,16 @@ const AuthPage = ({ uid, setUid }) => {
   };
 
   const handleSignIn = async () => {
```

Comment:
Maybe comment out the handleSignIn function, I am not really sure about this I think Alex is still testing something for the sign in part in the backend that does not use this?